packages/ui-avatar/src/Avatar/styles.js

```diff
@@ -98,15 +98,15 @@ const generateStyle = (
       ...variantStyles[shape]
     },
     initials: {
-      label: 'initials',
+      label: 'avatar_initials',
```

Comment:
The migration guide recommends double underscores: 'avatar__initials' to indicate BEM-like parent-child connection.

Reply:
Yapp, ty. Starting to get used to all the new rules